lnrpc/walletrpc/walletkit.proto

```diff
@@ -50,6 +50,11 @@ service WalletKit {
     */
     rpc NextAddr (AddrRequest) returns (AddrResponse);
 
+    /*
+    ListWalletAddresses returns all addresses in the underlying wallet.
+     */
+    rpc ListWalletAddresses (AddrRequest) returns (AddrsResponse);
```

Comment:
Would be very useful to have this as a command in `lncli wallet` as well (see `cmd/lncli/walletrpc_active.go`).

Reply:
Waiting for my local node to sync back up so I can test this, but I have it committed locally!